AppCenter/AppCenter/Internals/MSDependencyConfiguration.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@class MSHttpClient;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSDependencyConfiguration : NSObject
+
+/**
+ * Gets the http client.
+ *
+ * @return The http client.
+ */
++ (MSHttpClient *)getHttpClient;
```

Comment:
The intent was to be an interface, not a concrete class. We need to be able to switch to an alternate implementation in functional tests.